sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -217,6 +217,18 @@ def list_submitted_jobs(self, **kwargs):
         # type: (**Any) -> ItemPaged[JobStatusResult]
         """List all the submitted translation jobs under the Document Translation resource.
 
+        :keyword int top: indicates the total number of records the user wants to be returned across all pages.
+        :keyword int skip: indicates the number of records to skip from the list of batches based on the sorting
+            method specified. By default, we sort by descending start time.
+        :keyword int maxpagesize: is the maximum items returned in a page.  If more items are requested via top (or
+            top is not specified and there are more items to be returned), @nextLink will contain the link
+            to the next page.
+        :keyword list[str] ids: Ids to use in filtering.
+        :keyword list[str] statuses: Statuses to use in filtering.
+        :keyword ~datetime.datetime created_date_time_utc_start: the start datetime to get items after.
+        :keyword ~datetime.datetime created_date_time_utc_end: the end datetime to get items before.
```

Comment:
For convenience, we should accept a string datetime as well as the datetime.datetime type like we do in metrics advisor: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_client.py#L176

We can copy the `convert_datetime` helper method into our code, too. 

Also how do you feel about naming on this one? It feels a little long and not sure we need to include `utc` in the name. Maybe `jobs_created_before` and `jobs_created_after`?